### ISSUE TITLE
Обернуть плагин в asmdef

### ISSUE THREAD

**Issue by AndreyMishuta** [JustPlay-Max/PluginYG#7]
Если в проекте используются asmdef, то работать с плагином не получается. Достаточно сделать 2 asmdef(YandexGame.Runtime и YandexGame.Editor)

**Comment by JustPlay-Max** [comment]
Да, знаю о проблеме. Исправлять это буду в версии 2.0, если руки до нее дотянутся когда нибудь. Спасибо!
